rasp/base.py

```diff
@@ -45,6 +45,24 @@ def _session(self, *args, **kwargs):
         """
         return requests.session(*args, **kwargs)
 
+    @staticmethod
+    def as_func(cls, func_name):
+        """
+        Curries the "func_name" method
+
+        Returns:
+            User-defined class method
+        """
+        TmpEngine = deepcopy(cls)
+
+        method = None
+        class_name = TmpEngine.__class__.__name__
+        try:
+            method = getattr(TmpEngine, func_name)
```

Comment:
That's a good point! I'll update it. And sorry for playing fast and loose with the Docstring haha I'll get that fixed.